applications/SwimmingDEMApplication/python_scripts/hdf5_script.py

```diff
@@ -0,0 +1,70 @@
+# Import system python
+import os
+
+import numpy as np
+import h5py
+import KratosMultiphysics as Kratos
+from KratosMultiphysics import Vector
+import KratosMultiphysics.SwimmingDEMApplication as Dem
+
+class ErrorProjectionPostProcessTool(object):
```

Comment:
FYI I the hdf5application would probably work for you too

Reply:
> FYI I the hdf5application would probably work for you too

Yes, we want to migrate all our h5py python tools to it if possible. We will discuss it for a future PR